### PR TITLE
Deploy to starter machine with GH Actions

### PR DESCRIPTION
**Pull request by tsegismont**
See #143

### CODE REVIEW

**Comment by tsegismont** [comment]
@vietj I haven't configured the secret yet, I'll do it when this PR is merged

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -27,3 +29,28 @@ jobs:
...
+          java-version: 11
+      - name: Setup SSH
+        uses: kielabokkie/ssh-key-and-known-hosts-action@v1.1.0
+        with:
+          ssh-private-key: ${{ secrets.SSH_PRIVATE_KEY }}
+          ssh-host: start.vertx.io
+      - name: Assemble
+        run: ./gradlew assemble
+      - name: Run deployment script
+        run: bash .travis.deploy.bash
```
Comment (vietj) [inline .github/workflows/ci.yml:56]: I think we should remove the travis name here as we don't use it anymore.

**Review by vietj — APPROVED** [review]
LGTM, minor comment but it's not a biggie.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (tsegismont) [reply .github/workflows/ci.yml:56]: OK, I will update the filename and merge.

**Pull request merged**